apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -434,16 +434,16 @@ defmodule OMG.Eth.RootChain do
   end
 
   defp decode_deposit(log) do
-    non_indexed_keys = [:amount]
-    non_indexed_key_types = [{:uint, 256}]
+    non_indexed_keys = [:amount, :offset, :tokenids]
```

Comment:
I can't find `offset` elsewhere? is all code pushed out? where does this entry come from?

Reply:
I've added clarification https://github.com/hoardexchange/elixir-omg/commit/bbf706d152bd7ea8923a6eead4692ab66e28304c